packages/core/src/errors/validation-error.ts

```diff
@@ -172,11 +172,16 @@ export class ValidationErrorItem extends Error {
    *
    * @param useTypeAsNS controls whether the returned value is "namespace",
    *                    this parameter is ignored if the validator's `type` is not one of ValidationErrorItem.Origins
-   * @param NSSeparator a separator string for concatenating the namespace, must be not be empty,
-   *                    defaults to "." (fullstop). only used and validated if useTypeAsNS is TRUE.
    * @throws {Error}    thrown if NSSeparator is found to be invalid.
    */
   getValidatorKey(useTypeAsNS: false): string;
+
+  /**
+   * @param useTypeAsNS controls whether the returned value is "namespace",
+   *                    this parameter is ignored if the validator's `type` is not one of ValidationErrorItem.Origins
+   * @param NSSeparator a separator string for concatenating the namespace, must be not be empty,
+   *                    defaults to "." (fullstop). only used and validated if useTypeAsNS is TRUE.
+   */
```

Comment:
The reason that I did not do this before is that it does not copy over the rest of the explanation. Having the same text duplicated did not seem like the right solution to me. But maybe that's not something we can prevent.

Any reason why you did not copy over the rest of the text?

Reply:
Because the result on our TypeDoc looked "good enough" imo, but it differs between tools

I'd argue we could delete `useTypeAsNS` from the copy

![image](https://github.com/sequelize/sequelize/assets/1280915/eb5f0e50-65bf-40be-99a7-5e055c9bd434)

Reply:
I think it's good if we keep the useTypeAsNS in the copy, for the people that do not read more than that.